src/public/css/partials/footer.css

```
@@ -0,0 +1,111 @@
+footer {
+    background-color: #141718;
```

Los colores muevalos a una variable

Esto es solo un ejemplo:

```suggestion
    background-color: var(--dark);
```